sdk/cognitivelanguage/azure-ai-language-conversations/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 1.0.0b4 (2022-05-15)
+
+### Features Added
+* Conversation issue summarization task (Long-running operation)
+* Conversation PII extraction task (Long-running operation)
+
+### Breaking Changes
+* Client now uses python dictionaries for method parameters and results instead of classes.
+* Many input and result parameter name changes in `analyze_conversations()` method
```

Comment:
yeah! there was a bug introduced because of the newly added `cancel` api
i couldn't name the method in singular form, but i fixed that
will update changelog